shared/params.go

```diff
@@ -380,10 +380,22 @@ func ParseProductParam(r *http.Request) (product *Product, err error) {
 // It parses the 'products' parameter, split on commas, and also checks for the (repeatable)
 // 'product' params.
 func ParseProductsParam(r *http.Request) (ProductSpecs, error) {
-	productParams := ParseRepeatedParam(r, "product", "products")
+	repeatedParam := r.URL.Query()["product"]
+	pluralParam := r.URL.Query().Get("products")
+	// Replace nested ',' in the label part with a placeholder
+	nestedCommas := regexp.MustCompile("(\\[[^\\]]*),")
+	const comma = "%COMMA%"
```

Comment:
GitHub isn't happy with this string. It's trying to highlight it as a format string. I don't know whether it's just a code highlighting bug in GitHub. We could just use backticks instead of quotes.

Reply:
Done.